explainaboard/processors/text_to_sql_test.py

```diff
@@ -0,0 +1,14 @@
+"""Tests for explainaboard.processors.text_to_sql"""
+
+from __future__ import annotations
+
+import unittest
+
+from explainaboard.constants import TaskType
+from explainaboard.processors.processor_factory import get_processor_class
+from explainaboard.processors.text_to_sql import TextToSQLProcessor
+
+
+class QAOpenDomainProcessorTest(unittest.TestCase):
```

Comment:
```suggestion
class TextToSQLProcessorTest(unittest.TestCase):
```

Reply:
Oops. Thank you for pointing this out. I copied the function from the QA file but forgot the change the name. Have corrected it in the new commit.